lighthouse-core/gather/driver.js

```diff
@@ -279,11 +279,15 @@ class Driver {
    * Call protocol methods, with a timeout.
    * To configure the timeout for the next call, use 'setNextProtocolTimeout'.
    * @template {keyof LH.CrdpCommands} C
-   * @param {C} method
+   * @param {C|{method: C, sessionId: string}} methodObj
```

Comment:
Yah I dig this solution.

A followup PR can move `this._nextProtocolTimeout` into this methodObj. cc @Hoten

Reply:
I feel the same way about this as I did for #6347 :) We're trying to generalize a simple thing by making it more complicated, just for the two calls that need it.

Since they're both inside `driver`, would it be possible for now to make those two calls be calls of `_innerSendCommand` instead, leave `sendCommand` the same, and we setup a time to figure out what `driver` should look like in a multi target world (which we're going to need for SW work anyways)? I think long term that can also help with complexity when we start doing more opportunities with iframe resources, as ideally we can minimize the manual sessionId management that has to be done in gatherers.

Reply:
> Since they're both inside driver, would it be possible for now to make those two calls be calls of _innerSendCommand instead, leave sendCommand the same, and we set up a time to figure out what driver should look like in a multi target world (which is also going to help us do better SW work)?

sure